documentation/slate/source/includes/apis.md

```diff
@@ -137,24 +137,64 @@ paginate results.
 
 ### Parameters
 
+``` shell
+curl 'https://api.navitia.io/v1/coverage/sandbox/pt_objects?q=metro%201' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+HTTP/1.1 200 OK
+
+{
+    "pt_objects":[
+        {
+            "id":"line:RAT:M1",
+            "name":"RATP Metro 1 (Château de Vincennes - La Défense)",
+            "embedded_type":"line",
+            "line":{
+                "id":"line:RAT:M1",
+                "name":"Château de Vincennes - La Défense",
+                "code":"1",
+                "...": "..."
+            }
+        },
+        {
+            "id":"line:RAT:M11",
+            "name":"RATP Metro 11 (Mairie des Lilas - Châtelet)"
+            "embedded_type":"line",
+            "line":{
+                "...": "..."
+            },
+        },
+        {
+            "id":"line:RAT:M12",
+            "name":"RATP Metro 12 (Mairie d'Issy - Front Populaire)",
+            "embedded_type":"line",
+            "line":{
+                "...": "..."
+            }
+        },
+        {"...": "..."},
+        {"...": "..."}
+    ]
+}
+```
+
 #### <a name="depth"></a>depth
 
 This tiny parameter can expand Navitia power by making it more wordy.
 Here is some examples around "metro line 1" from the Parisian network:
 
 - Get "line 1" id
-	- <https://api.navitia.io/v1/coverage/fr-idf/pt_objects?q=metro%201>
+	- <https://api.navitia.io/v1/coverage/sandbox/pt_objects?q=metro%201>
 	- The id is "line:OIF:100110001:1OIF439"
 - Get routes for this line
-	- <https://api.navitia.io/v1/coverage/fr-idf/lines/line:OIF:100110001:1OIF439/routes>
+	- <https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/routes>
 - Want to get a tiny response? Just add "depth=0"
-	- <https://api.navitia.io/v1/coverage/fr-idf/lines/line:OIF:100110001:1OIF439/routes?depth=0>
+	- <https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/routes?depth=0>
 	- The response is lighter (parent lines disappear for example)
 - Want more informations, just add "depth=2"
-	- <https://api.navitia.io/v1/coverage/fr-idf/lines/line:OIF:100110001:1OIF439/routes?depth=2>
+	- <https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/routes?depth=2>
 	- The response is a little more verbose (with some geojson appear in response)
```

Comment:
We can see empty `geojson`, so I'll add a notice that we have geojson data in real data
``` json
"geojson": {
    "type": "MultiLineString",
    "coordinates": [ ]
}
```